ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -17,6 +17,7 @@
     <OutputPath>bin\x86\Debug\</OutputPath>
     <DefineConstants>DEBUG;NET46</DefineConstants>
     <DocumentationFile>bin\x86\Debug\ReactNative.Net46.XML</DocumentationFile>
+    <DebugType>full</DebugType>
   </PropertyGroup>
```

Comment:
I think I did it when I was debugging something and trying to get better information out of the debugger. I'll revert this and do a separate PR to make debug info consistent across the projects.